src/main/java/it/gov/pagopa/pu/debtpositions/util/Utilities.java

```diff
@@ -78,4 +78,12 @@ public static void checkImmutableField(String fieldName, Object original, Object
       modifiedFields.add(fieldName);
     }
   }
+
+  public static Double centsToEuro(Long cents) {
```

Comment:
never use double to store currency! you have not control on accuracy! Use BigDecimal instead

ps: we have many example, with relate tests to do this:
https://github.com/pagopa/idpay-reward-notification/blob/57f98daae2770dc2f707f8e4222c324171e59a3f/src/main/java/it/gov/pagopa/common/utils/CommonUtilities.java#L62